### PR TITLE
add support for list values

### PR DESCRIPTION
**Pull request by marlenepereira**

### CODE REVIEW

**Review by Niksko — CHANGES_REQUESTED** [review]
Looks good, but can we add a test for this please

**Comment by marlenepereira** [comment]
> Looks good, but can we add a test for this please

Added tests around setting  a`value` and `listValues`.

File: pkg/filters/set.go [hunk 1]
```diff
@@ -63,6 +63,11 @@ func (f *SetPackageFilter) kptfileSetterFilter() kio.Filter {
 			return node, nil
 		}
 
+		if len(f.ListValues) > 0 {
+			f.Value = f.ListValues[0]
+			f.ListValues = f.ListValues[1:]
+		}
```
Comment (ashlineldridge) [inline pkg/filters/set.go:69]: 🥇

**Review by ashlineldridge — APPROVED** [review]
Awesome work 👏🏼

**Review by Niksko — APPROVED** [review]
Great, looks good to me too!

**Pull request merged**